<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../../../../home/klyell/.designer/images/nmwrri_small_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Download Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloaddata">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
background-color: rgb(0, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Download Data</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>205</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startdate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="enddate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Choose start and end dates</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Choose site from dropdown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>710</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
background-color: rgb(0, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
background-color: rgb(0, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Choose table type from dropdown</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Sans Serif&quot;;color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>470</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Download data for site</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Download site information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadsite">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>550</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px; 
background-color: rgb(0, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Download Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/nmwrri_small_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>downloaddata</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>startdate</zorder>
  <zorder>enddate</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_4</zorder>
  <zorder>home</zorder>
  <zorder>update</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label_5</zorder>
  <zorder>error</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>downloadsite</zorder>
  <zorder>label_8</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
